eblocker-icapserver/src/main/java/org/eblocker/server/app/DeviceProperties.java

```diff
@@ -93,49 +103,49 @@ public DeviceProperties(@Named("deviceProperties") String devicePropertiesPath)
             LOG.warn("Cannot load device properties from {}, using default values", devicePropertiesPath, e);
         }
 
-        if (getSerialNumberPattern() != null) {
```

Comment:
When I added the @Nonnull, the IDE showed me that it no longer needs this if check, for example